tests/foreman/api/test_syncplan.py

```diff
@@ -178,7 +178,7 @@ def test_positive_create_with_sync_date(self):
                     sync_date=syncdate,
                 ).create()
                 self.assertEqual(
-                    syncdate.strftime('%Y-%m-%dT%H:%M:%SZ'),
+                    syncdate.strftime('%Y-%m-%d %H:%M:%S UTC'),
```

Comment:
@oshtaier nice catch! Double checked that - only returned format has changed, accepted one remains the same:
```python
>>> test = entities.SyncPlan(organization=1)
>>> test.create_missing()
>>> test.sync_date
datetime.datetime(2301, 10, 21, 10, 51, 34, 485312)
>>> datetime(2301, 10, 21, 10, 51, 34, 485312).strftime('%Y-%m-%d %H:%M:%S')
'2301-10-21 10:51:34'
>>> test = test.create()
>>> test.sync_date
u'2301-10-21 10:51:34 UTC'
```
Actually, API is smart enough to accept different formats, including the new one, but I don't think we need to update that - just because it's working ok right now and we'll need te re-test everything on different sat versions to be sure we won't introduce some regression.